tests/std/tests/P2510R3_text_formatting_pointers/test.cpp

```diff
@@ -0,0 +1,296 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <array>
+#include <cassert>
+#include <cctype>
+#include <charconv>
+#include <cstddef>
+#include <cwctype>
+#include <format>
+#include <functional>
+#include <ranges>
+
+#include <test_format_support.hpp>
+
+#define STR(Str) TYPED_LITERAL(CharT, Str)
+
+using namespace std;
+
+template <class CharT>
+basic_string<CharT> pointer_to_string(const void* ptr) {
+    char buffer[16];
+    auto [end, ec] = to_chars(buffer, buffer + 16, reinterpret_cast<uintptr_t>(ptr), 16);

```

Comment:
Similar nitpick - it would also be less confusing to avoid repeating 16 with two different meanings (16 hexits for 64-bit, base 16 always).